pkgs/development/python-modules/boto3/default.nix

```diff
@@ -4,47 +4,40 @@
 , botocore
 , jmespath
 , s3transfer
-, futures ? null
-, docutils
-, nose
-, mock
-, isPy3k
 }:
 
 buildPythonPackage rec {
   pname = "boto3";
-  version = "1.24.75"; # N.B: if you change this, change botocore and awscli to a matching version
+  version = "1.26.32"; # N.B: if you change this, change botocore and awscli to a matching version
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "sha256-e/oiQt+bH6c+F/TX959rIlYPVdXaLifleOWF725LFKc=";
+    hash = "sha256-4NYhUxOwPwmpo47MyIwdO6mGi8quuLIO622I/DAYuU0=";
   };
 
-  propagatedBuildInputs = [ botocore jmespath s3transfer ] ++ lib.optionals (!isPy3k) [ futures ];
-  checkInputs = [ docutils nose mock ];
+  propagatedBuildInputs = [
+    botocore
+    jmespath
+    s3transfer
+  ];
 
-  checkPhase = ''
-    runHook preCheck
-    # This method is not in mock. It might have appeared in some versions.
-    sed -i 's/action.assert_called_once()/self.assertEqual(action.call_count, 1)/' \
-      tests/unit/resources/test_factory.py
-    nosetests -d tests/unit --verbose
-    runHook postCheck
-  '';
-
-  # Network access
+  # boto3 Pypi package has no tests.
```

Comment:
Can we fetch from github instead?

Reply:
Thanks for the review.

I changed the boto3 package to fetch sources from Github directly so that we can run tests. But I had to disable integration tests because they require networking.